selftests/unit/test_result.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import argparse
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from avocado.core.result import Result
+
+
+class FakeNonUniqueIdJob(object):
```

Comment:
`FakeMissingUniqueIdJob` would probably reflect this more (non-unique-id in my understanding means that the id is there, but is not unique)